implementations.md

```diff
@@ -189,10 +189,10 @@ Various levels of support for UI generation primarily from the validation vocabu
     -   [JSON Editor](https://github.com/jdorn/json-editor) (MIT)
     -   [JSON Form (joshfire)](https://github.com/joshfire/jsonform) (joshfire) (MIT)
     -   [Json Forms (brutusin)](https://github.com/brutusin/json-forms) (brutusin) (MIT)
-    -   [JSONForms (jsonforms.io)](http://jsonforms.io) (EclipseSource) (MIT)
-    -   [Jsonary](http://jsonary.com/) (MIT)
+    -   [JSONForms (jsonforms.io)](https://jsonforms.io/) (EclipseSource) (MIT)
+    -   [Jsonary](https://github.com/jsonary-js/) (MIT)
```

Comment:
This should be https://github.com/jsonary-js/jsonary, not sure why the redirect of the old site went where it did.